lib/category-name-converter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
it looks like with pubcentral we're dis-allowing the use of `'use strict';` all together:

https://github.com/NBCUOTS/pubcentral/blob/f4037998d5bc1c8c8b86381e360c9cce4e24d129/.eslintrc#L39

unless there is an objection, I can just remove it from all files.